lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -105,7 +105,7 @@ function gutenberg_enqueue_global_styles() {
 }
 
 remove_action( 'wp_enqueue_scripts', 'wp_enqueue_global_styles' );
-remove_action( 'wp_footer', 'wp_enqueue_global_styles' );
+remove_action( 'wp_footer', 'wp_enqueue_global_styles', has_action( 'wp_footer', 'wp_enqueue_global_styles' ) );
```

Comment:
Can we hardcode priority here?

Reply:
For sure, I tested with `1` and it works. 

My reasoning was that it `has_action` gives us the right answer all the time. 

Is it bad practice or slow to query it?

Reply:
It was mostly a nitpick. I don't think I've seen `has_action` used this way, maybe only if priority is dynamically generated, and that's a bad idea for different reasons :)

Hardcoding the value should make scanning and getting all the data you need more accessible, IMO.